src/Sarif.Converters.UnitTests/ToolFormatConverterTests.cs

```diff
@@ -17,73 +20,82 @@ public class ToolFormatConverterTests
         private readonly ToolFormatConverter _converter = new ToolFormatConverter();
 
         [TestMethod]
-        [ExpectedException(typeof(ArgumentException))]
         public void ToolFormatConverter_ConvertToStandardFormat_DirectorySpecifiedAsDestination()
         {
             string input = Path.GetTempFileName();
             string directory = Environment.CurrentDirectory;
-            _converter.ConvertToStandardFormat(ToolFormat.AndroidStudio, input, directory);
+            Action action = () => _converter.ConvertToStandardFormat(ToolFormat.AndroidStudio, input, directory);
+
+            action.ShouldThrow<ArgumentException>();
```

Comment:
Several tests now use `ShouldThrow<T>` instead of `[ExpectedException]`. It's more readable, more flexible (for example, you can say `action.ShouldThrow<T>().WithMessage("*Something*");` to assert that the exception message mentions "Something"), and improves code coverage in your unit test code (which is important for reasons we can discuss).